### ISSUE TITLE
More precise kill messages

### ISSUE THREAD

**Issue by deathsythe47** [deathsythe47/base_entranced#56]
Not sure if this is possible serverside.

Some methods of death use the same messages, which is dumb. Would be nice if we had more. (see `cg_event.c`)

`MOD_CONC` and `MOD_CONC_ALT` use `KILLED_GENERIC` ("was killed by")....maybe should be "was obliterated by"

`MOD_CRUSH` uses `KILLED_GENERIC` ("was killed by")....maybe should be "was crushed by"

**Comment by deathsythe47** [comment]
I managed to get this working to a degree. I also added in midair and height checks to print special messages for airshots ("Player1 was rocket-aired by Player2"). Basically it involved using `trap_SendServerCommand` to simulate the obituary message and the "You killed Player2" message. However, you cannot properly simulate the death message for the victim ("Killed by Player2"), so I'm going to leave it out.
